server/routes/game.ts

```diff
@@ -0,0 +1,32 @@
+import express, { Request, Response } from "express";
+const router = express.Router();
+import { getGameHowLongToBeat, getGameRating } from "../utils";
+
+router.post("/", async (req: Request, res: Response) => {
```

Comment:
Kept everything as POST requests for ease of use, but could easily be GET requests since we're not depositing anything in code.